src/actions/deactivator.test.ts

```diff
@@ -55,12 +57,70 @@ beforeAll(() => {
   })
 })
 
-describe('deactivate from admin control mode', () => {
+describe('deactivate', () => {
+  test('should return wsman message if the message status is 401', async () => {
+    const clientObj = clientManager.getClientObject(clientId)
+    clientObj.uuid = deactivatemsg.payload.uuid
+    clientManager.setClientObject(clientObj)
+    const response = 'HTTP/1.1 401 Unauthorized\r\nWWW-Authenticate: Digest realm="Digest:8FF80F01A086065136E90CD6A08EBDB1", nonce="5bOx4i4TAAAAAAAAJ0IH1EXYw408gnUk",stale="false",qop="auth"\r\nContent-Type: text/html\r\nServer: Intel(R) Active Management Technology 15.0.23.1706\r\nContent-Length: 693\r\nConnection: close\r\n\r\n<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" >\n<html><head><link rel=stylesheet href=/styles.css>\n<meta http-equiv="Content-Type" content="text/html; charset=utf-8">\n<title>Intel&reg; Active Management Technology</title></head>\n<body>\n<table class=header>\n<tr><td valign=top nowrap>\n<p class=top1>Intel<font class=r><sup>&reg;</sup></font> Active Management Technology\n<td valign="middle"><img src="logo.gif" align="right" alt="Intel">\n</table>\n<br />\n<h2 class=warn>Log on failed. Incorrect user name or password, or user account temporarily locked.</h2>\n\n<p>\n<form METHOD="GET" action="index.htm"><h2><input type=submit value="Try again">\n</h2></form>\n<p>\n\n</body>\n</html>\n'
```

Comment:
You could probably make the header for 200 and 400 reusable and just differentiate the bodies?

Reply:
Resolved in deactivator PR